src/main/java/frc/robot/Constants.java

```diff
@@ -1,7 +1,25 @@
 package frc.robot;
+import edu.wpi.first.math.util.Units;
 
 public class Constants {
-
+	public static class VisionConstants {
+		public static final double CAM_HEIGHT_METERS = Units.inchesToMeters(20.75);
+		public static final double CAM_PITCH_RADIANS = Units.degreesToRadians(0);
+		public static final double CUBE_HEIGHT_METERS = Units.inchesToMeters(8.3);
+		public static final double CONE_HEIGHT_METERS = Units.inchesToMeters(12.8125);
+		public static final double HIGH_TAPE_HEIGHT_METERS = Units.inchesToMeters(43.75);
+		public static final double LOW_TAPE_HEIGHT_METERS = Units.inchesToMeters(24);
+		public static final double TAG_HEIGHT_METERS = Units.inchesToMeters(1.927);
+		public static final double ALIGNMENT_THRESHOLD_DEGREES = 3;
+		public static final int LOWERTAPE_PIPELINE_INDEX = 3;
```

Comment:
This would be a great place to use an [Enum](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html)

Reply:
Do you mean Enum for each of the pipelines?

Reply:
I'm thinking a PipelineIndex Enum, with values LOWERTAPE_PIPELINE, HIGHERTAPE_PIPELINE etc.